platooning_control/CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ ament_auto_add_executable(${node_exec}
 )
 
 # Register component
-rclcpp_components_register_nodes(${node_lib} "platoon_control::PlatoonControlPlugin")
+rclcpp_components_register_nodes(${node_lib} "platooning_control::PlatoonControlPlugin")
```

Comment:
sorry, not to cause extra work, but if the namespace is now platooning, shouldn't the class also be PlatooningControl?